test_app/settings.py

```diff
@@ -165,13 +165,15 @@
 SYSTEM_USERNAME = '_system'
 
 ANSIBLE_BASE_MANAGED_ROLE_REGISTRY = {
+    'sys_auditor': {'name': "Platform Auditor"},
     'team_member': {},
     'team_admin': {},
     'org_admin': {},
     'org_member': {},
     'cow_admin': {'shortname': 'admin_base', 'model_name': 'test_app.cow', 'name': 'Cow Admin'},
     'cow_moo': {'shortname': 'action_base', 'model_name': 'test_app.cow', 'name': 'Cow Mooer', 'action': 'say_cow'},
 }
+ANSIBLE_BASE_JWT_MANAGED_ROLES.append("System Auditor")  # noqa: F821 this is set by dynamic settings for jwt_consumer
```

Comment:
I'm not worried about this line, but it does seem pointless. We don't create this role definition in test_app. Some _individual tests_ might create it. It doesn't seem to make sense in test_app settings, but it's just test_app, so non-blocking comment.

Reply:
This is so other tests, which try to assign a role with the name `System Auditor` do not have their functionality interfered with.

I could just change everything to use `Platform Auditor` instead, but I think that should be addressed in a different PR.